### PR TITLE
update example generator

### LINKED ISSUE #21943: Nightly IT Example Generator workflow could be improved
Currently, Nightly IT Example Generator workflow has some problems:

1, Job has no timeout, there's [job]( https://github.com/apache/shardingsphere/actions/runs/3388544928/jobs/5630665471 ) cost more than 4 hours and it's still running:
图片

2, There're much failed jobs

3, There're `104` [jobs]( https://github.com/apache/shardingsphere/actions/runs/3388544928 ), every job might cost several minutes on `Build with Maven` (e.g. 5/7/8 minutes).

### PR DESCRIPTION
**Pull request by totalo**
Fixed #21943

### CODE REVIEW

**Review by taojintianxia — APPROVED** [review]

**Comment by taojintianxia** [comment]
the commit message is too casual.

File: examples/shardingsphere-example-generator/src/main/resources/template/jdbc/java/config/encrypt.ftl [hunk 1]
```diff
@@ -19,7 +19,7 @@
         Properties props = new Properties();
         props.setProperty("aes-key-value", "123456");
         EncryptColumnRuleConfiguration columnConfigAes = new EncryptColumnRuleConfiguration("phone", "phone", "", "", "phone_plain", "phone_encryptor", null);
-        EncryptColumnRuleConfiguration columnConfigTest = new EncryptColumnRuleConfiguration("status", "status", "assisted_query_status", "", "string_encryptor", "string_encryptor", null);
+        EncryptColumnRuleConfiguration columnConfigTest = new EncryptColumnRuleConfiguration("status", "status", "assisted_query_status", "", "","string_encryptor", "string_encryptor", null, null);
```
Comment (sandynz) [inline examples/shardingsphere-example-generator/src/main/resources/template/jdbc/java/config/encrypt.ftl:22]: About `"","string_encryptor"`, blank could be added after `,`

**Review by sandynz — CHANGES_REQUESTED** [review]

File: examples/shardingsphere-example-generator/src/main/resources/template/jdbc/java/config/encrypt.ftl (hunk 1 above)
Comment (totalo) [reply examples/shardingsphere-example-generator/src/main/resources/template/jdbc/java/config/encrypt.ftl]: done.

**Comment by totalo** [comment]
> the commit message is too casual.

Ok, I'll pay attention next time.

**Review by sandynz — APPROVED** [review]

**Pull request merged**